db/data/20210804083620_delete_zero_bps.rb

```diff
@@ -0,0 +1,16 @@
+class DeleteZeroBps < ActiveRecord::Migration[5.2]
+  def up
+    return unless CountryConfig.current[:abbreviation] == "ET"
+
+    zero_bps = BloodPressure.where(systolic: 0, diastolic: 0)
+
+    zero_bps.find_each do |blood_pressure|
+      blood_pressure.observation&.discard
```

Comment:
Should we also discard the encounter here?

Reply:
In theory, we'd check if there are any other activities associated with the encounter, and delete if nothing else is there. I chose not to make this thing that smart because all 0/0 BPs were imported from a spreadsheet where each element definitely represented a visit - ie, had a visit date, prescribed meds, etc. Hence I chose to leave the encounters alone. What do you think?

Reply:
Okay, makes sense to leave it as is in this case 👍